parlai/agents/transformer/modules.py

```diff
@@ -428,9 +430,9 @@ def forward_embedding(
             positions = (mask.cumsum(dim=1, dtype=torch.int64) - 1).clamp_(min=0)
         tensor = self.embeddings(input)
         if self.embeddings_scale:
-            tensor = tensor * np.sqrt(self.dim)
+            tensor = tensor * math.sqrt(self.dim)
 
-        if positions.max().item() > self.n_positions:
+        if positions.max().item() > self.n_positions and not torch.jit.is_scripting():
```

Comment:
Can we swap the is_scriptings so they happen first? I suspect that will also help with the modelparallel madness. 

They're technically a microoptimization here computed to doing this cpu() transfer

Reply:
Yes, changing